docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -661,3 +678,832 @@ For example, if you'd like to hide the sign up tab, you can override the `amplif
   `}</style>
   <Authenticator className="customization-hide-sign-up" />
 </Example>
+
+If you'd like to update the primary color of your submit button you can override the `amplify-button` class.
+
+```css
+.amplify-button[data-variation='primary'] {
+  background: linear-gradient(to right, #8360c3, #2ebf91);
```

Comment:
Can we demonstrate using the Amplify colors here instead of hex colors?
```suggestion
  background: linear-gradient(to right, var(AMPLIFY-COLOR-VARIABLE), var(AMPLIFY-COLOR-VARIABLE));
```

Reply:
Sure